sf-fx-runtime-java-runtime/src/main/java/com/salesforce/functions/jvm/runtime/invocation/undertow/UndertowInvocationInterface.java

```diff
@@ -165,6 +165,7 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
         String message = "Could not marshall function result: " + e.getCause().getMessage();
         makePlainTextErrorResponse(exchange, StatusCodes.BAD_REQUEST, message);
       } catch (SalesforceFunctionException e) {
+        e.printStackTrace();
```

Comment:
Is this leftover debugging or expected?

Reply:
This is an open issue that will be fixed separately, good catch though! :)